packages/web/src/utils/tempPlaylistId.ts

```diff
@@ -0,0 +1,11 @@
+export const getTempPlaylistId = () => {
+  // Minimum playlist ID, intentionally higher than legacy playlist ID range
+  const MIN_PLAYLIST_ID = 400000
+  // Maximum playlist ID, reflects postgres max integer value
+  const MAX_PLAYLIST_ID = 2147483647
```

Comment:
some concerns that i posted on @jowlee 's PR -- this might break our hashids

Reply:
replied https://github.com/AudiusProject/audius-protocol/pull/3706#discussion_r949451803